src/components/TodoList/Todo.tsx

```diff
@@ -0,0 +1,57 @@
+import classnames from 'classnames';
+import React from 'react';
+import { Loader } from '../Loader/Loader';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  isAdding: boolean,
+  todo: Todo,
+  deleteTodo:(param: number) => void,
+  loadingTodoIds: number[],
+};
+
+export const Todos: React.FC<Props> = ({
```

Comment:
Would be better to name the component **TodoItem**, because it is a single todo and to avoid confusing it with the type. Here, in the file name and in TodoList